ui/raidboss/data/04-sb/dungeon/ala_mhigo.ts

```diff
@@ -96,7 +96,7 @@ const triggerSet: TriggerSet<Data> = {
       alertText: (_data, _matches, output) => output.text!(),
       outputStrings: {
         text: {
-          en: 'Protean',
+          en: 'Protean (Spread Out)',
```

Comment:
Protean means something different to me than spread.  I understand that not everybody knows what this means, but I think it's better to call things consistently across multiple fights.  Folks can change it in the cactbot ui if they don't like it.

Reply:
“Protean”, however, is meaningless to people who didn’t raid Alex…

Reply:
I hear you, but I think there's not a better word for "spread out, but in relation to your angle to the boss" vs "spread out, in relation to each other" than "protean".  I think they are different and it's better to differentiate what the mechanic actually is and have a little confusion, and if folks want to call it something different then the they can use the cactbot ui to change it.